aaronpoweruser.ReadwiseUnofficial/src/NPReadwise.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import { insertContentUnderHeading } from "../../helpers/NPParagraph"
+import pluginJson from '../plugin.json'
+import { log, logDebug, logError, logWarn, clo, JSP } from '@helpers/dev'
+import { getOrMakeNote } from '@helpers/note'
+
+// This is the main function that will be called by NotePlan
+export async function readwiseSync(): Promise<void> {
+  const settings = DataStore.settings
+  const accessToken = settings.accessToken ?? ''
```

Comment:
I suggest a `showMessage(...)` to user if accessToken is '' or doesn't match the right length, and a `logError(...)` call as well.